k-distribution/tests/regression-new/issue-1789-rhsOr/1.test

```diff
@@ -0,0 +1 @@
+1
```

Comment:
Can we add a test for `0` as well, to make sure the `split` is working as intended @radumereuta ?